kratos/containers/distributed_system_vector.h

```diff
@@ -207,6 +207,15 @@ class DistributedSystemVector final
             AddEntry(*it);
     }
 
+    TDataType Norm() const
+    {
+        TDataType norm_squared = IndexPartition<TIndexType>(LocalSize()).template for_each< SumReduction<TDataType> >( [this](TIndexType i){
```

Comment:
similary to the distributed_csr here we should rely on the system_vector norm for the local part

Reply:
Agree. Nice.